dss/api/drs.py

```diff
@@ -0,0 +1,51 @@
+import requests
+
+from flask import jsonify, make_response, request
+
+from dss import dss_handler
+from dss.api.files import get_helper
+from dss.config import Replica
+from dss.util.version import datetime_from_timestamp
+
+
+@dss_handler
+def get_data_object(uuid: str):
+    """
+    Return a DRS data object dictionary for a given DSS file UUID and version.
```

Comment:
please include a reference to the DRS docs within the doc-string 
`https://ga4gh.github.io/data-repository-service-schemas/preview/release/drs-1.0.0/`